shared/viewmodels/RoomDirectoryViewModel.js

```diff
@@ -9,7 +9,7 @@ const HomeserverSelectionModalContentViewModel = require('matrix-public-archive-
 
 const DEFAULT_SERVER_LIST = ['matrix.org', 'gitter.im', 'libera.chat'];
 
-const ADDED_HOMESERVERS_LIST_LOCAL_STORAGE_KEY = 'addedHomservers';
+const ADDED_HOMESERVERS_LIST_LOCAL_STORAGE_KEY = 'addedHomeservers';
```

Comment:
This will result in added homeservers essentially being reset since we're reading and storing under a new key but we haven't even cut a release yet so it's fine ⏩